watertap/examples/flowsheets/ion_exchange/tests/test_ion_exchange_demo.py

```diff
@@ -92,80 +90,66 @@ def test_build_model(self, ix_0D):
+        assert m.fs.ion_exchange.config.isotherm == IsothermType.langmuir
 
         assert m.fs.ion_exchange.config.property_package is m.fs.properties
         assert "H2O" in m.fs.properties.component_list
 
-    @pytest.mark.requires_idaes_solver
     @pytest.mark.component
     def test_specific_operating_conditions(self, ix_0D):
         m = ix_0D
-        ixf.set_operating_conditions(m, feed_mass_frac=feed_mass_frac)
+        ixf.set_operating_conditions(m)
         ixf.initialize_system(m)
         assert degrees_of_freedom(m) == 0
 
         solver = get_solver()
         results = solver.solve(m)
         assert_optimal_termination(results)
         assert value(m.fs.feed.properties[0].flow_vol_phase["Liq"]) == pytest.approx(
-            0.05, abs=1e-6
+            0.05, abs=1e-3
         )
         assert value(m.fs.product.properties[0].flow_vol_phase["Liq"]) == pytest.approx(
-            0.049901566, abs=1e-6
+            0.049995010, abs=1e-3
         )
-        assert value(m.fs.regen.properties[0].flow_vol_phase["Liq"]) == pytest.approx(
-            0.016666, abs=1e-6
-        )
-        assert value(
-            sum(
-                m.fs.feed.properties[0].conc_equiv_phase_comp["Liq", j]
-                for j in m.fs.properties.ion_set
-            )
-        ) == pytest.approx(5, rel=1e-3)
         assert value(
             sum(
-                m.fs.product.properties[0].conc_equiv_phase_comp["Liq", j]
+                m.fs.feed.properties[0].conc_mass_phase_comp["Liq", j]
                 for j in m.fs.properties.ion_set
             )
-        ) == pytest.approx(0.01057086, rel=1e-3)
+        ) == pytest.approx(0.1, rel=1e-3)
         assert value(
             sum(
-                m.fs.regen.properties[0].conc_equiv_phase_comp["Liq", j]
+                m.fs.product.properties[0].conc_mass_phase_comp["Liq", j]
                 for j in m.fs.properties.ion_set
             )
-        ) == pytest.approx(471.99797, rel=1e-3)
+        ) == pytest.approx(0.000211438, rel=1e-3)
 
-        assert value(m.fs.ion_exchange.number_columns) == pytest.approx(4, rel=1e-6)
-        assert value(m.fs.ion_exchange.bed_depth) == pytest.approx(1.7, rel=1e-6)
+        assert value(m.fs.ion_exchange.number_columns) == pytest.approx(4, rel=1e-3)
+        assert value(m.fs.ion_exchange.bed_depth) == pytest.approx(1.7, rel=1e-3)
         assert value(m.fs.ion_exchange.t_breakthru) == pytest.approx(
-            56759.75759, rel=1e-6
+            56759.75759, rel=1e-3
         )
         assert value(m.fs.ion_exchange.partition_ratio) == pytest.approx(
-            235.99899, rel=1e-6
+            235.99899, rel=1e-3
         )
 
         assert value(m.fs.costing.specific_energy_consumption) == pytest.approx(
-            0.057352, abs=0.001
+            0.057245, abs=0.001
         )
-        assert value(m.fs.costing.LCOW) == pytest.approx(0.219, abs=0.01)
+        assert value(m.fs.costing.LCOW) == pytest.approx(0.222437, abs=1e-3)
```

Comment:
BTW= do you `assert_units_consistent` anywhere in this test?

Reply:
Yup, for all three frames: lines 243, 520, 821.